Sources/Client/Model/Pagination.swift

```diff
@@ -139,13 +138,14 @@ public enum PaginationOption: Encodable, Equatable, Hashable {
     
     public static func == (lhs: Self, rhs: Self) -> Bool {
         switch (lhs, rhs) {
-        case (.limit, .limit),
-             (.offset, .offset),
-             (.greaterThan, .greaterThan),
-             (.greaterThanOrEqual, .greaterThanOrEqual),
-             (.lessThan, .lessThan),
-             (.lessThanOrEqual, .lessThanOrEqual):
-            return true
+        case (.limit(let value1), .limit(let value2)),
+             (.offset(let value1), .offset(let value2)):
+            return value1 == value2
+        case (.greaterThan(let value1), .greaterThan(let value2)),
+             (.greaterThanOrEqual(let value1), .greaterThanOrEqual(let value2)),
+             (.lessThan(let value1), .lessThan(let value2)),
+             (.lessThanOrEqual(let value1), .lessThanOrEqual(let value2)):
+            return value1 == value2
```

Comment:
This is actually changing logic, does it affect anywhere?
Why don't we group all cases, instead of separating `limit` and `offset`?

Reply:
I think it was a bug.
`.limit` and `.offset` has `Int` value type and others `String`.